src/main/webapp/app/drug/drug.scss

```diff
@@ -0,0 +1,13 @@
+.drug-select {
+    max-width: 813px;
```

Comment:
lol 813?

Reply:
That's because other default input boxes max-width is 813px. I will set max-width for all input box together in the future.